app/spell_check.py

```diff
@@ -40,7 +41,71 @@ def check_other_lang(input_list, word_list, dictionary):
     results = []
     for word_idx in word_list:
         if not regex.search(r'([^\p{L}\'\-])', input_list[word_idx]):
-            lower_word = str(unicodedata.normalize('NFC', input_list[word_idx])).lower()
+            lower_word = toLower(input_list[word_idx])
```

Comment:
Is this working with Irish diacritics properly?

Reply:
yep, I now realize that is not the built in lower function in python, sorry it was too early when I looked at this the first time apparently